docs/src/main/asciidoc/_admin-guide/subsystem-configuration/Metrics.adoc

```diff
@@ -0,0 +1,105 @@
+[[MicroProfile_Metrics_SmallRye]]
+= Metrics Subsystem Configuration
+
+[NOTE]
+====
+This subsystem exposes only base metrics from WildFly Management Model and JVM MBeans.
+
+Support for https://microprofile.io/project/eclipse/microprofile-metrics[Eclipse MicroProfile Metrics] is provided by
+the _microprofile-metrics-smallrye_ subsystem.
+====
+
+[[required-extension-metrics]]
+== Extension
+
+This `org.wildfly.extension.metrics` extension is included in all the standalone configurations included in the
+WildFly distribution as well as the `metrics` layer.
+
+You can also add the extension to a configuration without it either by adding
+an `<extension module="org.wildfly.extension.metrics"/>`
+element to the xml or by using the following CLI operation:
+
+[source,options="nowrap"]
+----
+[standalone@localhost:9990 /] /extension=org.wildfly.extension.metrics:add
+----
+
+== Management Model
+
+The `/subsystem=metrics` resource defines three attributes:
+
+* `security-enabled` - a boolean to indicate whether authentication is required to access the HTTP metrics endpoint (described below). By default, it is `true`. The
+standalone configurations explicitly sets it to `false` to accept unauthenticated access to the HTTP endpoints.
+* `exposed-subsystems` - a list of strings corresponding to the names of subsystems that exposes their metrics in the HTTP metrics endpoints.
+By default, it is not defined (there will be no metrics exposed by subsystem. The special wildcard `*` can be used to expose metrics from _all_ subsystems. The standalone
+configuration sets this attribute to `*`.
+* `prefix` - A string to prepend to WildFly metrics that are exposed by the HTTP endpoint `/metrics` with the Prometheus output format.
+
+[[metrics-http-endpoint]]
+== HTTP Endpoint
+
+The Metric HTTP endpoint is accessible on WildFly HTTP management interface http://localhost:9990/metrics[http://localhost:9990/metrics].
+
+Secured access to the HTTP endpoint is controlled by the `security-enabled` attribute of the `/subsystem=metrics` resource.
+If it is set to `true`, the HTTP client must be authenticated.
+
+If the application server is healthy, it will return a `200 OK` response:
```

Comment:
I'm not sure if 'healthy' is a good word here. If I'm thinking about metrics and hear 'health' I think in terms of what a liveness/readiness probe would return, which I *think* is not what you mean here. There may have been boot errors etc that would reflect poor 'health' but would not result in a non-200 from /metrics.

Reply:
I agree. That looks like a bad copy/paste from the Microprofile-health guide.